docs/guide/appearances.md

```diff
@@ -73,45 +73,6 @@ BrushedAluminium {
 
 - `IBLStrength`: Defines the strength of ambient lighting from the Cubemap node.
 
-## CarLeather
```

Comment:
This should not be edited manually.

Reply:
Thank you, I just applied the script and it seems to work smoothly.